### ISSUE TITLE
Change text-color of active cell in dark-mode

### ISSUE THREAD

**Issue by ttowntom** [NoroffFEU/yeetsheet#87]
Due to accessibility reasons the text color of active text in dark mode should be changed to uphold WCAG compatability.
